### PR TITLE
Add package readmes

### PR DESCRIPTION
**Pull request by michaelgsharp**
Update to use preview 5 for building.
Fixed some minor formatting issues that came from the preview 5 update.
Added package readmes to 5 more packages for now, #7195

### CODE REVIEW

**Comment by michaelgsharp** [comment]
This will also fix all the blockers for #7165 except the async test one. I'll get a pr up for that one.

**Comment by michaelgsharp** [comment]
Actually it was such a minor change I just added it to this PR as well.

File: src/Microsoft.ML.FastTree/PACKAGE.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+## About
+
+ML.NET is a cross-platform open-source machine learning framework which makes machine learning accessible to .NET developers.
+
```
Comment (tarekgh) [inline src/Microsoft.ML.FastTree/PACKAGE.md:4]: Usually the read me list the major types and small specific sample in the package readme. Are we using the same readme template used in the runtime repo? .

**Review by tarekgh — APPROVED** [review]
Left comment/question. LGTM otherwise.

File: src/Microsoft.ML.OnnxTransformer/PACKAGE.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+* [API documentation](https://learn.microsoft.com/en-us/dotnet/api/microsoft.ml.transforms.onnx)
+
+## Related Packages
+
+* Main Package: [Microsoft.ML](https://www.nuget.org/packages/Microsoft.ML)
+* Core data abstraction: [Microsoft.ML.DataView](https://www.nuget.org/packages/Microsoft.ML.DataView)
+* LightGBM Model Support: [Microsoft.ML.LightGbm](https://www.nuget.org/packages/Microsoft.ML.LightGbm)
+* Fast Tree: [Microsoft.ML.FastTree](https://www.nuget.org/packages/Microsoft.ML.FastTree)
+* Image analytics: [Microsoft.ML.ImageAnalytics](https://www.nuget.org/packages/Microsoft.ML.ImageAnalytics)
+* Reccomender: [Microsoft.ML.Recommender](https://www.nuget.org/packages/Microsoft.ML.Recommender)
```
Comment (LittleLittleCloud) [inline src/Microsoft.ML.OnnxTransformer/PACKAGE.md:38]:
```suggestion
* Recommender: [Microsoft.ML.Recommender](https://www.nuget.org/packages/Microsoft.ML.Recommender)
```

Comment (michaelgsharp) [reply src/Microsoft.ML.OnnxTransformer/PACKAGE.md]: Fixed in all of them.

File: src/Microsoft.ML.FastTree/PACKAGE.md (hunk 1 above)
Comment (michaelgsharp) [reply src/Microsoft.ML.FastTree/PACKAGE.md:4]: So it looks like FastTree is the only one that I missed for the major types. I'll get those in. As far as samples go, we have a link to the code samples repo. Do you think thats enough or do you think we should actually put some samples in here? I basically just copied what had been done for the main ML.NET package to these others ones.

**Pull request merged**